connectivity_summary.py

```diff
@@ -46,7 +52,10 @@ def arg_to_date(arg):
     print("No experiments loaded; bailing out.")
     sys.exit(-1)
 
-expts = all_expts.select(start=args.start, stop=args.stop, region=args.region)
+for i, ex in enumerate(all_expts._expts):
```

Comment:
I would make this:

    for i, ex in enumerate(all_expts):
        ex.summary_id = i

The main reason is that putting an underscore at the beginning of a name (`_expts` in this case) means that you are not supposed to access that attribute from outside the class.